cunit/annotate.testc

```diff
@@ -2212,7 +2213,7 @@ static int set_up(void)
     mboxlist_init();
     mboxlist_open(NULL);
 
-    struct mboxlock *namespacelock = mboxname_usernamespacelock(MBOXNAME1_INT);
+    unslock_t *unslock = unslock_lockmb(MBOXNAME1_INT);
```

Comment:
I may be the odd one out, but all I can see here is `un-slock`.  If nothing else, a `_` might go a long way.

Reply:
> While we are it, I would prefer this even less opaquely named:
> 
> `user_nslock` rather than `unslock` would go a long way with:
> 
>     1. it's consistent with using the file name of the header that defines this function as a prefix (`user.h` in this case)
> 
>     2. it's clearer to me to understand it's a user namespace lock when looking at the function name

Agreed